Source/Core/DolphinQt2/Settings.h

```diff
@@ -17,12 +18,12 @@ enum class Language;
 class InputConfig;
 
 // UI settings to be stored in the config directory.
-class Settings final : public QSettings
+class Settings final : public QSettings, public NonCopyable
```

Comment:
Ah yeah, that makes sense. I thought about that, but am so used to always writing `public` by default.